stream-approvals/approvals-bootstrap-task/pom.xml

```diff
@@ -15,19 +15,39 @@
     <packaging>jar</packaging>
     <name>Stream :: Approvals Bootstrap Task</name>
 
+    <properties>
+        <checkstyle.disable.checks>true</checkstyle.disable.checks>
```

Comment:
it seems a mistake because we should not disable the checkstyle right?

Reply:
Yes. In a perfect world you are absolutely right. But it also fails for all existing code and not new code. So if you do want checkstyle enabled, we need to ensure all existing code is documented